pkg/autodetect/main.go

```diff
@@ -105,6 +116,21 @@ func (a *autoDetect) HPAVersion() (AutoscalingVersion, error) {
 	return AutoscalingVersionUnknown, errors.New("Failed to find apiGroup autoscaling")
 }
 
+func (a *autoDetect) OpenShiftRoutesAvailability() (OpenShiftRoutesAvailability, error) {
+	apiList, err := a.dcl.ServerGroups()
+	if err != nil {
+		return OpenShiftRoutesUnknown, err
+	}
+
+	for _, apiGroup := range apiList.Groups {
+		if apiGroup.Name == "route.openshift.io" {
```

Comment:
Why is the api group name here different?

Reply:
This code checks if the `route.openshift.io` API resource is installed in the cluster.